src/components/Calculator.js

```diff
@@ -1,51 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 import './Calculator.css';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-  }
+function Calculator() {
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you make use of ES6 syntax for your functions.

## EXAMPLE
```
const Calculator = () => (`

);
```